Miscellaneous fixes (#4)
pulsar-edit/docs: docs/getting-started/adding-terminal-commands.md

* Add these lines to your shell startup file:

    ```shell
    alias pulsar="Pulsar.AppImage"
    alias ppm="Pulsar.AppImage -p"
    ```

* Restart your shell and make sure your new aliases work:

    ```
    pulsar --version
    ppm --version
    ```

## Windows

:::danger
**TODO:** Certain parts of these instructions won’t work right until [#1054](https://github.com/pulsar-edit/pulsar/pull/1054) is landed.
:::

Windows installations of Pulsar can attempt to place `pulsar` and `ppm` in your path via the settings. Select _File > Settings_, choose the _System_ pane, and select “Add Pulsar to PATH.”

If this command runs properly, you should be set — but we’ll make sure in a second. If not, you’ll have to add them manually.

### Checking your path

Open a terminal application — Windows Terminal is available for free from the Microsoft Store, or you can use Windows’ built-in PowerShell — and run:

```
Get-Command pulsar
Get-Command ppm
```

If these commands are identified and point to paths within your Pulsar installation directory, you’re done. If not, keep reading.

### Manually modifying your path

:::tip

> Before you attempt this, you should remind yourself where you installed Pulsar. If you installed it system-wide, for instance, its installation directory would’ve defaulted to `C:\Program Files\Pulsar`; but if you installed it for just the current user, then it’ll live somewhere deep within that user’s home folder.

Reviewer: ```suggestion
Before you attempt this, you should remind yourself where you installed Pulsar. If you installed it system-wide, for instance, its installation directory would’ve defaulted to `C:\Program Files\Pulsar`; but if you installed it for just the current user, then it'll default to `C:\Users\(my-user)\AppData\Local\Programs\pulsar`.
```

Our Windows installer should always be installing to this directory (when preforming a user install). As well as this is the default installation directory, so I'd want to make sure we list it, as that's likely where most users have Pulsar installed.
